GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -21,20 +22,28 @@
 
 namespace GooglePlayInstant.Editor
 {
-    //TODO: Resolve conversation about upload vs setting path
     /// <summary>
     /// Class that generates Unity loading scenes for instant apps.
     /// </summary>
     public class PlayInstantLoadingScreenGenerator
     {
         public const string LoadingSceneName = "play-instant-loading-screen-scene";
 
-        public static string loadingScreenImagePath;
-        
-        //TODO: add documentation
+        private const string LoadingScreenJsonFileName = "LoadingScreenConfig.json";
+
+        private static readonly string LoadingScreenScenePath =
+            Path.Combine("Assets", "PlayInstantLoadingScreenAssets");
+
+        private static readonly string LoadingScreenJsonPath = Path.Combine(LoadingScreenScenePath, "Resources");
+
+        public static string LoadingScreenImagePath;
```

Comment:
- Public so needs documentation.
- Does this need to be publicly settable?
- Could be a property with a getter only?
- Does not need to be persisted, correct?